churchland_pipeline/acquisition.py

```diff
@@ -184,15 +190,16 @@ def populate(self, monkey, rig, task):
                         Session.User.insert1((date,monkey,'emt2177'))
 
                     # insert notes
-                    notesIdx = [re.search('.*notes\.txt',x) for x in sessFiles]
-                    if any(notesIdx):
-
-                        notesFile = sessFiles[list(compress(range(len(notesIdx)), notesIdx))[0]]
+                    try:
+                        notesFile = next(x for x in sessFiles if re.search('.*notes\.txt',x))
                         fid = open(raw_path + date + '/' + notesFile,'r')
```

Comment:
This portion should be in the `else` clause of the `try` statement block.  The `try` block should hug the statement raising the exception.